### ISSUE TITLE
Unable to load react-dom

### ISSUE THREAD

**Issue by tbranyen** [tbranyen/babel-plugin-transform-commonjs#6]
Modules like React DOM have conditional exports based on `if` statements in the source. This is similar, but distinct from how React loads itself.

We need to support:

``` js
if (hasNativePerformanceNow) {
  var Performance = performance;
  exports.unstable_now = function () {
    return Performance.now();
  };
} else {
  exports.unstable_now = function () {
    return localDate.now();
  };
}
```

We can do this by making the binding mutable with `export let` and then reassigning the export inside the source.

``` js
export let unstable_now = null;

if (hasNativePerformanceNow) {
  var Performance = performance;
  unstable_now = function () {
    return Performance.now();
  };
} else {
  unstable_now = function () {
    return localDate.now();
  };
}
```

**Comment by tbranyen** [comment]
This is solvable much easier than I originally though. Keeping the same dynamics as loading React itself. I don't think this needs a mutable binding.

```js
if (hasNativePerformanceNow) {
  var Performance = performance;
  unstable_now = function () {
    return Performance.now();
  };
} else {
  unstable_now = function () {
    return localDate.now();
  };
}

export const unstable_now = exports.unstable_now;
```

Although, perhaps making all bindings mutable by default is more spec-complaint.

**Comment by tbranyen** [comment]
Closed in e44fc6b
